activities/communication/communications-plan.md

```diff
@@ -25,102 +25,198 @@ We believe it’s more important to talk concretely about what we’ve already d
-* offers codebase stewardship for different projects
-* offers productization and marketing of the codebases
-* actually provides guidance and support to help public organizations make their code open
-* isn’t tied to the EU project lifecycle
-* has a reproducible process suitable not just for cities, but all public organizations
-* is international, not national
+Success is shown these responses:
 
-We are also actively building our ecosystem by:
+* being asked to join a conversation
+* Github issues/contributions to our tools
+* recommendations or referrals
+* joining our events
+* endorsing/hyping our events/co*hosting our events
+* invitations to speak at/contribute to their platforms (publications and events)
 
-* connecting members with like-minded parties (members and other public organizations, affiliates and structural funders)
-* boosting awareness of public code
-* setting standards and creating reusable resources (by providing thought leadership!)
-* demonstrating the value of public code with well documented, public business cases and risk analyses
-* nurturing our community online, and – where sensible – in person
+### Public organisation leadership
+
+Our goal for engagement with this group is:
+
+* they're aware of us and our mission, and believe this is the right thing to do
+* they believe we can add value to their organizations
+* we're not an organizational risk to them
+* they think public code is achievable in their organization
+* they're willing to invest political capital because they think will add to their careers (good optics?)
+
+Success is shown these responses:
+
+* request to prepare something for the leader
+* being asked to join a conversation
+* request for hard copy of our tools (Standard, Governance Game, etc)
+* recommendations or referrals
+* invitations to speak at/contribute to their platforms (publications and events)
+
+### Middle management
+
+Our goal for engagement with this group is:
+
+* they're aware of us and our mission, and believe this is the right thing to do
+* they believe we can add value to their organizations* we're not an organizational risk to them
+* they think public code is achievable in their organization
+
+Success is shown these responses:
+
+* spontaneous recommendations or referrals
+* us being asked to join a conversation
+* requests for our tools in their preferred format/language
+* request for 1:1 contact
+* successfully delivered 1:1 contact
+* codebase specific consultation
+* invitations to speak at/contribute to their platforms (publications and events)
+* references in gov't policy/strategy documents
+
+### Think tanks (influential civil society organizations with intellectual output)
+
+Our goal for engagement with this group is:
+
+* they're aware of us and our mission
+* think we're doing valuable work well
+* want them to see us as a key part of solving the problem
+* public recommendations and referrals from them
```

Comment:
This is good/interesting, and I think should be a two-way street - if we expect others to recommend/refer us, we should be ready to do the same for them

Reply:
Apart from me disagreeing with this since we're looking to be more of a tool for others than an influencer I also think that this is a consideration that is not part of being a goal.

I'm going to resolve this, if you disagree, feel free to reopen.